src/main/java/com/google/flourbot/api/Bot.java

```diff
@@ -79,9 +80,17 @@ public void onEvent(@RequestBody JsonNode event) throws IOException, GeneralSecu
         String threadId = event.at("/message/thread/name").asText();
         
         ChatResponse chatResponse = macroExecutionModule.execute(email, message, threadId);
+        
         replyText = chatResponse.getReplyText();
-        Card card = CardResponse.createCardResponse(replyText);
-        reply.setCards(Collections.singletonList(card));
+        ActionType actionType = chatResponse.getActionType();
+        String documentUrl = chatResponse.getDocumentUrl();
```

Comment:
What is `documentUrl` here?

Reply:
sheetUrl so it creates a button to open up the doc in browser basically

Reply:
Oh okay neat!